components/02-elements/tab/tab.hbs

```diff
@@ -16,7 +14,12 @@
              "
              data-content="{{ tabId }}"
         >
-            {{{ content }}}
+            {{#if content}}
+                {{{ content }}}
+            {{/if}}
+            {{#if contentElement}}
+                {{ render (component contentElement) contentContext }}
+            {{/if}}
         </div>
     {{/each}}
 </div>
```

Comment:
as we discussed on chat, we must add `if` statement with this script b/c if we have more than one it will not work in components. Then the last `tab` should have variable set to true

Reply:
Added `if` to script tag